tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/visibility/retry/RetryVisibilityTest.java

```diff
@@ -264,19 +263,19 @@ private void checkServiceCall(int nbExpectedRetries, RetryService service, Strin
             service.service();
             Assert.fail(String.format("in %s#%s service() should have failed",
                     RetryVisibilityTest.class.getSimpleName(), testName));
-        } catch (IOException re) {
+        } catch (TestException re) {
             Assert.assertEquals(
                     service.getNumberOfServiceCalls(),
                     expectedNbCalls,
                     String.format("in %s#%s service() should have been called exactly %d times",
                             RetryVisibilityTest.class.getSimpleName(),
                             testName,
                             expectedNbCalls));
-        } catch (RuntimeException ex) {
+        } catch (TestException ex) {
```

Comment:
You've changed this test to use `TestException` rather than `IOException` which is fine, but this second catch block still needs to change to catch either `Exception` or `RuntimeException`, since this catch block is dealing with _unexpected_ exceptions.

Reply:
I'm actually surprised that having two catch blocks for the same exception type isn't a compile error.